docs/integrations/gitops/overview.mdx

```diff
@@ -27,7 +27,7 @@ Adding tests, benchmarks, code-reviews, and adding more manual code - provide yo
 
 ### Setting up Gitops
 
-The feature is available to all Permit users as a self-service. Please follow the guide **[here](/integrations/gitops/github)**.
+The feature is available **in trial** to all Permit users as a self-service. Please follow the guide **[here](/integrations/gitops/github)**.
```

Comment:
change to EAP (early access program)

Reply:
I literally copied what Or asked to write here - https://permit-inc.slack.com/archives/C01RDD9BUTD/p1711458754854169